plugins/org.eclipse.n4js.n4jsx.generator/plugin.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright (c) 2016 NumberFour AG.
+All rights reserved. This program and the accompanying materials
+are made available under the terms of the Eclipse Public License v1.0
+which accompanies this distribution, and is available at
+http://www.eclipse.org/legal/epl-v10.html
+
+Contributors:
+  NumberFour AG - Initial API and implementation
+-->
+<?eclipse version="3.4"?>
+<plugin>
+   <!-- Register ECMAScript generator for N4JSX -->
+   <extension
+         point="org.eclipse.n4js.generator.common.subgenerator">
```

Comment:
This seems to be the left-over registration of the subgenerator without extension factory. (As just discussed @qtran-n4)

Reply:
removed.